palantirnet/drupal-10-development
# TECH-53: Add PHP Compatibility coding standard to PHPCS sniffs

Adds the PHP Compatibility coding standard sniff to `ddev check` and `ddev sniff`. 

To test:

- Spin up the project
- `ddev clone crop` (or another module)
- `ddev check crop`
  - Ensure that PHPCS has run with PHP Compatibility checker.
- `ddev sniff crop`
  - Ensure that PHPCS has run with PHP Compatibility checker.
